velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -753,6 +753,9 @@ class PlanBuilder {
   ///  rows between a + 10 preceding and 10 following)"
   PlanBuilder& window(const std::vector<std::string>& windowFunctions);
 
+  /// StreamingWindowBuild will be used to calculate the window partitions.
```

Comment:
Use active verbs and clarify correct usage of this API.

```
Adds WindowNode to compute window functions over pre-sorted inputs.

All functions must use same partition by and sorting keys and input must be already sorted on these.
```

Reply:
Updated.